### DISCUSSION TITLE
[아티클] SF Symbols에 애니메이션 적용하기

### DISCUSSION THREAD

**Discussion by yeniful** [DeveloperAcademy-POSTECH/.github#76] [General]
작년부터 SF Symbols에 애니메이션을 적용할 수 있게된 것을 아시나요?
특히나 토글적인 버튼을 구현하고 표시할 때 유용할 것 같아요 !
관심있는 분들 해당 예제 아티클 읽어보시면 좋을 것 같습니다-!

🔗 링크
https://www.donnywals.com/animating-sf-symbols-on-ios-18/?utm_source=substack&utm_medium=email

**Comment by Seon-U** [comment]
대박, 완전 신기합니다. 좋은 글 감사해요! :)

**Reply by yeniful** [reply]
댓글 감사해요!! 앞으로도 재밌어 보이는 글 자주 들고올게요 ㅎㅎ!!
